tachyon/math/finite_fields/quartic_extension_field_unittest.cc

```diff
@@ -0,0 +1,208 @@
+#include <optional>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/math/finite_fields/baby_bear/baby_bear4.h"
+#include "tachyon/math/finite_fields/test/finite_field_test.h"
+
+namespace tachyon::math {
+
+namespace {
+
+using F4 = BabyBear4;
+using F = BabyBear;
+
+class QuaticExtensionFieldTest : public FiniteFieldTest<F4> {};
+
+}  // namespace
+
+TEST_F(QuaticExtensionFieldTest, Zero) {
+  EXPECT_TRUE(F4::Zero().IsZero());
+  EXPECT_FALSE(F4::One().IsZero());
+}
+
+TEST_F(QuaticExtensionFieldTest, One) {
+  EXPECT_TRUE(F4::One().IsOne());
+  EXPECT_FALSE(F4::Zero().IsOne());
+}
+
+TEST_F(QuaticExtensionFieldTest, Random) {
+  bool success = false;
+  F4 r = F4::Random();
+  for (size_t i = 0; i < 100; ++i) {
+    if (r != F4::Random()) {
+      success = true;
+      break;
+    }
+  }
```

Comment:
Are we testing based on probability? 🤔 Is this the regular way to test "Random"?

Reply:
Hmm.. yes it depends on possibility.. I am not sure how we can test random properly..

Reply:
Hmm I suppose for now, since our Random tests are implemented this way, and the probability of this test failing is really low, we can keep it this way. I can't think of other ways to test this besides seeds, but that would take time to implement as well. Please resolve this once you read it!